posts/privilege_bingo.md

```diff
@@ -0,0 +1,53 @@
+# Privilege bingo
+
+Intersectionality, BAME, cis, anonymous hiring, inclusion. These words are likely not on your ticket if you are playing privilege bingo. Like a canary in a coal mine, a lack of understanding of these concepts will likely mean your daily actions reinforce systemic discrimination.
+
+We aren't good at diversity and that is something we need to work on as a business, as an industry, and as a collection of humans. There are direct and indirect effects caused by our policies and our privilege that actively work against us building a more diverse team.
+
+We didn't set out to avoid diversity, we just didn't and perhaps still don't have enough knowledge to encourage it. We didn't implement a refer a friend scheme to discourage diverse hiring, however it's fairly well known that your friends, especially ex-colleagues will mostly be like you in terms of gender and ethnicity given you're in an industry already suffering from a lack of diversity.
+
+## Awakening from ignorance
+
+Ignorance is bliss right? I'm unsure that's right but it's certainly easier. Through slow osmosis, from forgiving friends offering words of advice, and by growing noise about diversity, we are slowly waking up as an organisation.
+
+> The first step toward change is awareness. The second step is acceptance.
+>
+> ~ Nathaniel Brandon
+
+The first step has to be awareness of the fact we're poor at diversity. I mean it's hard to argue looking at the fact we're almost all white guys. The next step is acceptance, a willingness to come to terms with reality without protest or reaction. Lastly, action.
+
+## Identifying smells
+
+Coming back to the refer a friend scheme, why did we implement that scheme? We actually did it to encourage more of the same types of people joining. It's not that we wanted to encourage more of the same ethnicity, gender, or other intersections. We did it because we needed to hire quickly, and hiring people recommend by those we already know and trust is easier than the alternatives.
+
+With this ease, comes a problem. The issue is that there are direct consequences of enacting a policy like this. You will immediately be discouraging diversity by inviting more of the same gender and ethnicity unless you have previously worked in diverse organisations. You are also rewarding and reinforcing this kind of behaviour in your organisation.
+
+Smells are often a little difficult to work out, especially being so naive to privilege. Detecting the smells of exclusionary practices, much like code smells, gets easier with experience I am sure.
+
+## Accepting failures
+
+The tougher part is to gain wider support. Coming back to the privilege bingo, when a team isn't educated in areas of diversity, it is difficult to make diversity a high priority. Sure, diversity is gaining wider attention and people are talking the talk, but walking the walk is a whole other thing.
```

Comment:
On this, I've found that people are ignorant, ill-informed, or against diversity. So often I encounter "there's nothing we can do", "it's not worth the effort", "women just don't want to work with computers", or "women will end up having babies so not worth employing them as a small business".

It's only through me looking into intersectional feminism have I started understanding these concepts. I think there's a huge education piece to be done here. Also, there is a growing collection of resources for small tech businesses to start focussing on this issue, but it still feels like the early days. Are there resources out there that we are completely missing?

I think relevance is also a big thing here, the whole "its not our problem" or "Im not sure what we can do" type attitudes certainly play a big part in this.

Do you think I should change these paragraphs to place more responsibility on our need to accept its our problem and we're the ones causing it? Is there really a wealth of information on how to tackle this out there?